Home/Home.html

```diff
@@ -1,13 +1,165 @@
+        <div class="title">
+            <p id="welcome">
+                "Welcome to GamerZportal"
+            </p>
+            <p id="desc_2">
+                Game Forum 2023
+            </p>
+        </div>
+        <div class="description">
+            <p>
+                Here you can find the latest update of games and mods and etc etc. Lorem ipsum dolor sit, amet
+                consectetur adipisicing elit. Quis quo ex sapiente maiores totam ab iure laboriosam culpa doloremque
+                deleniti minus, ratione eaque explicabo esse autem quaerat accusantium aliquam fuga?
+            </p>
+        </div>
+        <div class="date">
+            <div class="part1">Gettogether - 2015.10.15(THU) ~ 2015.10.18(SUN)</div>
+            <div class="part2">@National Museum of korea</div>
+        </div>
+    </section>
+    <section class="MainProgram">
+        <div class="title1">
+            Main Programs
+        </div>
+        <hr>
+        <div class="programs">
+            <div class="Program">
+                <img src="/images/fg.png" alt="">
+                <p class="classtitle-1">
+                    Featured Games
+                </p>
+                <p class="desc-3">
+                    Highlight popular or trending games that are generating a lot of interest.
+                </p>
+            </div>
+            <div class="Program">
+                <img src="/images/wt.jpg" alt="">
+                <p class="classtitle-1">
+                    Latest Walkthroughs
+                </p>
+                <p class="desc-3">
+                    Showcase recently published walkthroughs for newly released or popular games.
+                </p>
+            </div>
+            <div class="Program">
+                <img src="/images/gr.jpeg" alt="">
+                <p class="classtitle-1">
+                    Game Reviews
+                </p>
+                <p class="desc-3">
+                    Provide summaries and ratings for games, helping users make informed decisions.
+                </p>
+            </div>
+            <div class="Program">
+                <img src="/images/gc.png" alt="">
+                <p class="classtitle-1">
+                    Game Categories
+                </p>
+                <p class="desc-3">
+                    Organize games into genres or categories (e.g., action, adventure, puzzle) for easy browsing
+                </p>
+            </div>
+            <div class="Program">
+                <img src="/images/spotlight_3174373.png" alt="">
+                <p class="classtitle-1">
+                    Community Spotlights
+                </p>
+                <p class="desc-3">
+                    Showcase user-generated content such as fan art, cosplay, or impressive gameplay videos.
+                </p>
+            </div>
+        </div>
+
+        <button id="Jp">Join the program</button>
+    </section>
+    <section class="Sponsors">
```

Comment:
- [ ] Please properly implement the featured speakers on mobile version as shown in the project design below 👍 
- project design
![mfs](https://github.com/Rdzaman20000/GamersPortal/assets/89055502/ec5775f9-659d-465a-87d0-3d53806ca14f)
- your design
![mfss](https://github.com/Rdzaman20000/GamersPortal/assets/89055502/cb848439-870f-48ce-bdf9-80fc33857303)